src/utils/one-of.ts

```diff
@@ -0,0 +1,3 @@
+export const oneOf = (values: string[], match: string) => {
+    return values.some(value => value === match);
```

Comment:
Seems simpler to inline the check.

Reply:
It may be simpler, but this does a lot for readability and the performance cost of invoking a function is miniscule. Unless it leads to an explicit performance issue, I prefer to optimize for readability.